src/content/docs/it/concepts/islands.mdx

```diff
@@ -1,66 +1,87 @@
+Con le isole, lo JavaScript lato client viene caricato solo per i componenti interattivi espliciti che contrassegni utilizzando le direttive `client:*`.
+
+E poiché l'interazione è configurata a livello di componente, puoi gestire diverse priorità di caricamento per ogni componente in base al suo utilizzo. Ad esempio, `client:idle` dice a un componente di caricarsi quando il browser diventa inattivo e `client:visible` dice a un componente di caricarsi solo una volta che entra nel viewport.
 
 ## Quali sono i vantaggi delle isole?
 
-Il vantaggio più evidente nell'utilizzo delle Isole Astro è la performance: la maggior parte del tuo sito web viene convertito in HTML statico veloce e JavaScript viene caricato solo per i singoli componenti che ne hanno bisogno. JavaScript è uno degli asset più lenti da caricare per byte, quindi ogni byte conta.
+Il vantaggio più ovvio nella creazione con le Isole Astro è la prestazione: la maggior parte del tuo sito web viene convertita in HTML e CSS veloci e statici e lo JavaScript viene caricato solo per i singoli componenti che ne hanno bisogno. Lo JavaScript è uno degli asset più lenti che puoi caricare per byte, quindi ogni byte conta.  
 
-Un altro vantaggio è il caricamento parallelo. Nell'illustrazione di esempio sopra, l'isola "carosello di immagini" a bassa priorità non deve bloccare l'isola "intestazione" ad alta priorità. Le due vengono caricate in parallelo e idratate in isolamento, il che significa che l'intestazione diventa immediatamente interattiva senza dover aspettare il carosello più pesante più in basso nella pagina.
+Un altro vantaggio è il caricamento parallelo. Nell'esempio di illustrazione sopra, l'"isola carosello immagini" a bassa priorità non deve bloccare l'"isola intestazione" ad alta priorità. I due vengono caricati in parallelo e idratati in isolamento, il che significa che l'intestazione diventa interattiva immediatamente senza dover attendere il carosello di immagini più pesante più in basso nella pagina.  
 
-Ancora meglio, puoi dire ad Astro esattamente come e quando renderizzare ciascun componente. Se quel carosello di immagini è davvero pesante da caricare, puoi aggiungere una [direttiva client](/it/reference/directives-reference/#client-directives) speciale che indica ad Astro di caricare il carosello solo quando diventa visibile sulla pagina. Se l'utente non lo vede mai, non viene mai caricato.
+Ancora meglio, puoi dire ad Astro esattamente come e quando renderizzare ogni componente. Se quel carosello di immagini è davvero costoso da caricare, puoi allegare una [direttiva client](/it/reference/directives-reference/#direttive-client) speciale che dice ad Astro di caricare il carosello solo quando diventa visibile nella pagina. Se l'utente non lo vede mai, non viene mai caricato.
```

Comment:
```suggestion
Ancora meglio, puoi dire ad Astro esattamente come e quando renderizzare ogni componente. Se quel carosello di immagini è davvero costoso da caricare, puoi allegare una [direttiva client](/it/reference/directives-reference/#client-directives) speciale che dice ad Astro di caricare il carosello solo quando diventa visibile nella pagina. Se l'utente non lo vede mai, non viene mai caricato.
```

Reply:
I addressed this in [this comment](https://github.com/withastro/docs/pull/6837#issuecomment-1934898705), do you think it should not be translated in the meantime?

Reply:
Leave it as it is. Every time we merge something into the repo a maintainer should update the PR in order to run all checks to safely merge. If I do it or someone else should first merge the PR who has no problems to be merged.

Reply:
The change suggested by @dreyfus92 is required before merging this PR. Then when the directives reference page is updated, that PR can include an update also to this page to use `#direttive-client`.